### PR TITLE
DPS Documentation

### PR DESCRIPTION
**Pull request by ondrejtomcik**

### CODE REVIEW

File: content/en/device-provisioning-service/overview.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+deactivate dps
+deactivate d
+d -> hub: Connect and authenticate using Identity Certificate
+activate d
+
+@enduml
+{{< /plantuml >}}
+
+1. The device contacts the DPS endpoint configured by the device application or by a tool which discovered the device or preset at the factory. Devices proves it's identity using a Manufacturer Certificate or TPM's endorsment key.
+2. The DPS finds DOS finds the matching Enrollment Group.
```
Comment (jkralik) [inline content/en/device-provisioning-service/overview.md:78]: DOS ? - what is it ?

File: content/en/device-provisioning-service/overview.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+note over op, hub
+Enrollment Entry including Devices key identifiers was created
+end note
+d -> dps: Connect and authenticate
+activate d
+dps -> dps: Find matching Enrollment Group
+activate dps
+d -> dps: Send Identity Certificate Signing Request
+d -> dps: Retrieve ACL configuration
+d -> d: Device ownership transfered
```
Comment (jkralik) [inline content/en/device-provisioning-service/overview.md:66]: This is 0 step. Device is owned by DPS client.

File: content/en/device-provisioning-service/overview.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+d -> dps: Get access token for plgd hub API
```
Comment (jkralik) [inline content/en/device-provisioning-service/overview.md:67]: this is the last step -> we don't want to change resources during connecting to the cloud.

File: content/en/device-provisioning-service/overview.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+keywords: [provisioning, TPM, manufacturer-certificate]
+menu:
+  docs:
+    parent: device-provisioning-service
+    weight: 50
+toc: true
+---
+
+
+Provisioning milions of devices in a secure and scalable manner without reqrequiering any human interaction is what our plgd Device Provisioning Service (DPS) is capable of. Doesn't matter if you're running the plgd hub on-premise, off-premise or a fully managed instance, the DPS makes sure it trusts only your devices and provision them to the right plgd hub instance. Exactly no human interaction, zero-touch to provision and pre-configure milions of devices just-in-time and securely, using manufacturer certificates or TPMs.
```
Comment (Danielius1922) [inline content/en/device-provisioning-service/overview.md:16]: reqrequiering -> requiring (just run the whole thing through spellchecker)

**Pull request merged**